src/modules/iotjs_module_buffer.c

```diff
@@ -174,6 +174,22 @@ size_t iotjs_bufferwrap_copy(iotjs_bufferwrap_t* bufferwrap, const char* src,
   return iotjs_bufferwrap_copy_internal(bufferwrap, src, 0, len, 0);
 }
 
+static size_t bufferslice_idx_setter(int64_t point, size_t len) {
+  size_t idx;
+  if (point < 0) {
+    if ((size_t)(-point) > len) {
+      idx = SIZE_MAX;
+    } else {
+      idx = (size_t)point + len;
+    }
+  } else {
+    idx = (size_t)point;
+  }
+
+
+  idx = bound_range(idx, 0, len);
+  return idx;
```

Comment:
Just directly return with the bound_range value: `return bound_range(idx, 0, len);`